mix.exs

```diff
@@ -46,12 +46,18 @@ defmodule Boombox.Mixfile do
   defp deps do
     [
       {:membrane_core, "~> 1.1"},
-      {:membrane_webrtc_plugin, "~> 0.22.0"},
+      {
+        :membrane_webrtc_plugin,
+        github: "membraneframework/membrane_webrtc_plugin",
+        branch: "support-various-video-codecs-in-source"
```

Comment:
seems like this needs some attention https://github.com/membraneframework/membrane_webrtc_plugin/pull/11